### PR TITLE
Bug 1579189

### PR DESCRIPTION
**Pull request by gkaihorodova**

### CODE REVIEW

**Comment by rcritten** [comment]
PEP-8 errors:
./ipatests/test_integration/test_external_ca.py:141:41: E128 continuation line under-indented for visual indent
./ipatests/test_integration/test_external_ca.py:142:41: E128 continuation line under-indented for visual indent
./ipatests/test_integration/test_external_ca.py:143:41: E128 continuation line under-indented for visual indent
./ipatests/test_integration/test_external_ca.py:144:41: E128 continuation line under-indented for visual indent
./ipatests/test_integration/test_external_ca.py:145:41: E128 continuation line under-indented for visual indent
./ipatests/test_integration/test_external_ca.py:145:64: E231 missing whitespace after ','
./ipatests/test_integration/test_external_ca.py:146:41: E128 continuation line under-indented for visual indent
./ipatests/test_integration/test_external_ca.py:147:41: E128 continuation line under-indented for visual indent
./ipatests/test_integration/test_external_ca.py:148:41: E128 continuation line under-indented for visual indent
./ipatests/test_integration/test_external_ca.py:149:41: E128 continuation line under-indented for visual indent

**Comment by tiran** [comment]
``./ipatests/test_integration/test_external_ca.py:146:36: E231 missing whitespace after ','``

File: ipatests/test_integration/test_external_ca.py [hunk 1]
```diff
@@ -136,6 +136,20 @@ def test_external_ca(self):
...
         tasks.install_replica(self.master, self.replicas[0])
 
+        # check that nsds5ReplicaReleaseTimeout option was set
+        result = self.master.run_command([
+            'ldapsearch',
+            '-x',
+            '-D',
+            'cn=directory manager',
+            '-w', self.master.config.dirman_password,
+            "'-b', 'cn=mapping tree', 'cn=config'",
```
Comment (tiran) [inline ipatests/test_integration/test_external_ca.py]: ``'-b', 'cn=mapping tree,cn=config',``

**Comment by tiran** [comment]
@gkaihorodova Test passed, finally! :)

Please remove the temp commit.

**Comment by gkaihorodova** [comment]
@tiran awesome! 
Thank you for your assistance!

**Comment by tiran** [comment]
master:

* 84ae625fe2c3786f7c5430f23a55c171ff54e110 check nsds5ReplicaReleaseTimeout option was set

**Pull request closed without merging**